.github/pr_issue_template.md

```diff
@@ -35,7 +36,6 @@ Example how to mark a checkbox :-
 - [ ] I have made corresponding changes to the documentation.
 - [ ] My changes generate no new warnings.
 - [ ] I have added tests that prove my fix is effective or that my feature works.
-- [ ] Any dependent changes have been merged and published in downstream modules.
```

Comment:
Why did you remove this @Sloth-Panda?

Reply:
sir, I was confused about what you told so removed that I will be adding that

Reply:
@Sloth-Panda Please put the line back. I was saying your PR was not following that (when "you" wrote the rules, "you" weren't following them for your PR, that's what I meant). Is it clear now? 😅 Sorry for any confusion caused.

Reply:
Yes, I did get that